src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class PopulationResponse(CamelCaseModel):
+    label: str = Field(..., title="Data reference label")
+    source: str = Field(..., title="Data source name")
+    value: int = Field(..., title="The population value")
+    updated: str = Field(..., title="Data updated at timestamp text")
+
+
+class PopulationRequest(CamelCaseModel):
+    # country: Optional[str] = Field(None, title="Country")
+    city: Optional[str] = Field(
+        "",
```

Comment:
If you define the field as optional, then I think it'd make better sense to set the default to `None` (equals null in JSON) rather than an empty string. Could possibly even be nice to require it either has a length or is None.